src/Packetery/Module/Checkout/CheckoutSettings.php

```diff
@@ -0,0 +1,248 @@
+			'weight'                     => $widgetWeight,
+			'carrierConfig'              => $carriersConfigForWidget,
+			'isCarDeliverySampleEnabled' => $this->carDeliveryConfig->isSampleEnabled(),
+			'isAgeVerificationRequired'  => $this->cartService->isAgeVerification18PlusRequired(),
+			'pickupPointAttrs'           => Order\Attribute::$pickupPointAttrs,
+			'homeDeliveryAttrs'          => Order\Attribute::$homeDeliveryAttrs,
+			'carDeliveryAttrs'           => Order\Attribute::$carDeliveryAttrs,
+			'carDeliveryCarriers'        => Entity\Carrier::CAR_DELIVERY_CARRIERS,
+			'expeditionDay'              => $this->getCarDeliveryExpeditionDay(),
+			'appIdentity'                => Plugin::getAppIdentity(),
+			'packeteryApiKey'            => $this->optionsProvider->get_api_key(),
+			'widgetAutoOpen'             => $this->optionsProvider->shouldWidgetOpenAutomatically(),
+			'saveSelectedPickupPointUrl' => $this->apiRouter->getSaveSelectedPickupPointUrl(),
+			'saveValidatedAddressUrl'    => $this->apiRouter->getSaveValidatedAddressUrl(),
+			'saveCarDeliveryDetailsUrl'  => $this->apiRouter->getSaveCarDeliveryDetailsUrl(),
+			'removeSavedDataUrl'         => $this->apiRouter->getRemoveSavedDataUrl(),
+			'adminAjaxUrl'               => $this->wpAdapter->adminUrl( 'admin-ajax.php' ),
+			'nonce'                      => $this->wpAdapter->createNonce( 'wp_rest' ),
+			'savedData'                  => $this->storage->getFromTransient(),
+			'translations'               => [
+				'packeta'                       => $this->wpAdapter->__( 'Packeta', 'packeta' ),
+				'choosePickupPoint'             => $this->wpAdapter->__( 'Choose pickup point', 'packeta' ),
+				'pickupPointNotChosen'          => $this->wpAdapter->__( 'Pickup point is not chosen.', 'packeta' ),
+				'placeholderText'               => $this->wpAdapter->__( 'Loading Packeta widget button...', 'packeta' ),
+				'chooseAddress'                 => $this->wpAdapter->__( 'Choose delivery address', 'packeta' ),
+				'addressValidationIsOutOfOrder' => $this->wpAdapter->__( 'Address validation is out of order', 'packeta' ),
+				'invalidAddressCountrySelected' => $this->wpAdapter->__( 'The selected country does not correspond to the destination country.', 'packeta' ),
+				'deliveryAddressNotification'   => $this->wpAdapter->__( 'The order will be delivered to the address:', 'packeta' ),
+				'addressIsNotValidatedAndRequiredByCarrier' => $this->wpAdapter->__( 'Delivery address has not been chosen. Choosing a delivery address using the widget is required by this carrier.', 'packeta' ),
+			],
+		];
+	}
+
+	/**
+	 * Used to provide additional settings for blocks checkout.
+	 *
+	 * @return void
+	 */
+	public function actionCreateSettingsAjax(): void {
+		$settings = [];
+		if ( $this->wcAdapter->cart() instanceof WC_Cart ) {
+			$settings['isAgeVerificationRequired'] = $this->cartService->isAgeVerification18PlusRequired();
+		}
+
+		$this->wpAdapter->sendJson( $settings );
```

Comment:
je třeba vůbec to posílat v případě, že je to prázdné pole?

Reply:
je to ideální, aby na straně Reactu nebyly zbytečné ify, nebo chyby v konzoli